log.graphqls

```diff
@@ -34,6 +34,7 @@ type Log {
     statusCode: String
     contentType: ContentType!
     content: String
+    tags: [LogTag!]
```

Comment:
Where is the definition of `LogTag`?

Reply:
https://github.com/apache/skywalking-query-protocol/blob/494a86ddfafd6d38de4f8932ce8213d1ed3c2e89/log.graphqls#L66-L68